src/common/components/ResponsiveImage/index.tsx

```diff
@@ -1,16 +1,88 @@
-import React, { ImgHTMLAttributes } from 'react';
+import React, { FC, ImgHTMLAttributes, useEffect, useState } from 'react';
+
+import { useBoundingRect } from 'common/hooks/useBoundingRect';
+import { getKeys } from 'common/utils/tsHacks';
+
 import { DOMAIN } from '../../constants/endpoints';
-import IResponsiveImage, { IResponsiveImageSizes } from '../../models/ResponsiveImage';
+import IResponsiveImage, {
+  IMAGE_DIMENTIONS,
+  IResponsiveImageSizes,
+  ResponsiveImageTypes,
+} from '../../models/ResponsiveImage';
```

Comment:
https://github.com/dotkom/onlineweb-frontend/wiki/(Opinionated)-Structureing-and-ordering-of-imports

Reply:
> Any file not residing in the same module as the current file should be imported with a root relative route.
>
> It may be a good thing to import as root relative if the path looks like this as well: ../../../models/Something; But I won't strictly condone that.